### PR TITLE
feat: tab support moreIcon

### PR DESCRIPTION
**Pull request by zty42**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/29695

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Tabs support moreIcon |
| 🇨🇳 Chinese | Tabs 新增 moreIcon 参数 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/index.zh-CN.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | centered | 标签居中展示 | boolean | false | 4.4.0 |
 | defaultActiveKey | 初始化选中面板的 key，如果没有设置 activeKey | string | `第一个面板` |  |
 | hideAdd | 是否隐藏加号图标，在 `type="editable-card"` 时有效 | boolean | false |  |
+| moreIcon | 自定义折叠 icon | ReactNode | <EllipsisOutlined /> |  |
```
Comment (afc163) [inline components/tabs/index.zh-CN.md:34]: English document

Comment (tianyuan233) [reply components/tabs/index.zh-CN.md:34]: added ~

File: components/tabs/index.en-US.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Ant Design has 3 types of Tabs for different situations.
 | centered | Centers tabs | boolean | false | 4.4.0 |
 | defaultActiveKey | Initial active TabPane's key, if `activeKey` is not set | string | - |  |
 | hideAdd | Hide plus icon or not. Only works while `type="editable-card"` | boolean | false |  |
+| moreIcon | custom icon of ellipsis | ReactNode | <EllipsisOutlined /> |  |
```
Comment (xrkffgg) [inline components/tabs/index.en-US.md:31]:
```suggestion
| moreIcon | The custom icon of ellipsis | ReactNode | <EllipsisOutlined /> | 4.14.0  |
```

File: components/tabs/index.zh-CN.md (hunk 1 above)
Comment (xrkffgg) [inline components/tabs/index.zh-CN.md:34]:
```suggestion
| moreIcon | 自定义折叠 icon | ReactNode | <EllipsisOutlined /> | 4.14.0 |
```

**Comment by xrkffgg** [comment]
PR 里的 changelog 填一下

**Comment by tianyuan233** [comment]
填啦

**Comment by tianyuan233** [comment]
> PR 里的 changelog 填一下

填啦

File: components/tabs/index.zh-CN.md [hunk 3]
```diff
@@ -31,6 +31,7 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | centered | 标签居中展示 | boolean | false | 4.4.0 |
 | defaultActiveKey | 初始化选中面板的 key，如果没有设置 activeKey | string | `第一个面板` |  |
 | hideAdd | 是否隐藏加号图标，在 `type="editable-card"` 时有效 | boolean | false |  |
+| moreIcon | 自定义折叠 icon | ReactNode | <EllipsisOutlined /> | 4.14.0 |
```
Comment (xrkffgg) [inline components/tabs/index.zh-CN.md:34]:
```suggestion
| moreIcon | 自定义折叠 icon | ReactNode | &lt;EllipsisOutlined /> | 4.14.0 |
```

File: components/tabs/index.en-US.md [hunk 4]
```diff
@@ -28,6 +28,7 @@ Ant Design has 3 types of Tabs for different situations.
 | centered | Centers tabs | boolean | false | 4.4.0 |
 | defaultActiveKey | Initial active TabPane's key, if `activeKey` is not set | string | - |  |
 | hideAdd | Hide plus icon or not. Only works while `type="editable-card"` | boolean | false |  |
+| moreIcon | The custom icon of ellipsis | ReactNode | <EllipsisOutlined /> | 4.14.0  |
```
Comment (xrkffgg) [inline components/tabs/index.en-US.md:31]:
```suggestion
| moreIcon | The custom icon of ellipsis | ReactNode | &lt;EllipsisOutlined /> | 4.14.0  |
```

**Review by xrkffgg — APPROVED** [review]

**Pull request merged**